### PR TITLE
Adding of the navigation bar component.

### LINKED ISSUE #12: make navbar component
add items like `cube`, `card`, `register`, `login`

### PR DESCRIPTION
**Pull request by alisa-alisa**
Resolves #12 
Adding navigation bar component.
Adding resources file.
Fixing tests/adding new tests/removing not useful tests.

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -36,15 +39,18 @@ const firebaseConfig = {
...
   ],
   imports: [
     BrowserModule,
     AppRoutingModule,
     BrowserAnimationsModule,
     AngularFireModule.initializeApp(firebaseConfig),
     AngularFireAuthModule, // auth
-    AngularFirestoreModule // firestore
+    AngularFirestoreModule, // firestore
+    MatToolbarModule,
```
Comment (endqwerty) [inline src/app/app.module.ts:52]: does material angular have an import all?

File: src/string-resources.ts [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export class Constants {
```
Comment (endqwerty) [inline src/string-resources.ts:1]: can you name this file en-us? that way we can just swap out language files in the future?

File: src/app/app.module.ts (hunk 1 above)
Comment (alisa-alisa) [reply src/app/app.module.ts:50]:
```
import { MatToolbarModule } from '@angular/material/toolbar';
import { MatButtonModule } from '@angular/material/button';
```

Yes

Comment (alisa-alisa) [reply src/app/app.module.ts:50]: Sorry didn't understand correctly. Not that I am aware of.

File: src/string-resources.ts (hunk 2 above)
Comment (alisa-alisa) [reply src/string-resources.ts:1]: I can. Basically I was investigating different ways of making localization, but it seems just a separate file is fine for now. I will rename

File: src/app/app.component.html [hunk 3]
```diff
@@ -1,2 +1 @@
-<div automation=base-template>Hi</div>
-<router-outlet></router-outlet>
```
Comment (alisa-alisa) [inline src/app/app.component.html:2]: I will add it back once I need it) probably with the next review. For now, it is not really used 🤔

**Review by endqwerty — APPROVED** [review]

**Pull request merged**